python/torch_mlir/eager_mode/torch_mlir_dispatch.py

```diff
@@ -0,0 +1,258 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+import re
+import warnings
+from typing import Any
+from typing import List, Dict
+
+import numpy as np
+import torch
+import torch._C
+from torch.utils._pytree import tree_map
+from torch_mlir._mlir_libs._mlir.passmanager import PassManager
+
+from torch_mlir.dialects.torch.importer.jit_ir.build_tools.registry import Registry
```

Comment:
This Registry class should only be used by build tools, not production code. Can you write your code to not use it? Doesn't PyTorch have a way to introspect the schema directly?

Reply:
you can get `is_mutable` from the schema object but the alias info doesn't have python bindings:

```
>>> schema.arguments[0].alias_info

Traceback (most recent call last):
  File "/home/mlevental/miniconda3/envs/torch-mlir/lib/python3.9/site-packages/IPython/core/interactiveshell.py", line 3251, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-15-b9d9b5a0259d>", line 1, in <module>
    schema.arguments[0].alias_info
AttributeError: 'torch._C.Argument' object has no attribute 'alias_info'
```